examples/deploy/addon/helloworld/resources/helloworld_clustermanagementaddon.yaml

```diff
@@ -9,3 +9,13 @@ spec:
   supportedConfigs:
   - group: addon.open-cluster-management.io
     resource: addondeploymentconfigs
+  installStrategy:
```

Comment:
It seems to need annotation `addon.open-cluster-management.io/lifecycle: addon-manager` to make installStrategy work, or wait for https://github.com/open-cluster-management-io/ocm/pull/374.